lib/infra/infra-stack.ts

```diff
@@ -5,35 +5,35 @@ The OpenSearch Contributors require contributions made to
 this file be licensed under the Apache-2.0 license or a
 compatible open source license. */
 
+import {
```

Comment:
This is just reorder of all the imports right?

Reply:
Yes my IDE does that!